src/Tribe/Aggregator/Records.php

```diff
@@ -76,6 +101,17 @@ public function filter_edit_link( $link, $post, $context ) {
 		return Tribe__Events__Aggregator__Page::instance()->get_url( $args );
 	}
 
+	/**
+	 * Adjusting the "Delete Post" link.
+	 *
+	 * @since 4.3.0
+	 *
+	 * @param string $link    The delete link.
+	 * @param int    $post    The post ID.
+	 * @param bool   $context Whether to bypass the Trash and force deletion. Default false.
```

Comment:
@codingmusician This param should actually be `$force_delete` and not `$context`. It is not used anywhere in the method so it would be safe to change. Should it be changed?

Reply:
I don't see that it's necessary to change the variable name. You're right. It would make more sense to use something more descriptive like `$force_delete`, but let's leave that for if/when this method is ever updated. Good question!